java/src/server/facade/ServerFacade.java

```diff
@@ -93,8 +102,9 @@ public void addAI(int gameID, Object aiType) throws AddAIException {
      * @throws ListAIException
      */
     @Override
-    public void listAI() throws ListAIException {
-
+    public ListAIDTO listAI(int gameId) throws ListAIException {
+        Game game = gameManager.getGameByID(gameId);
+        return new ListAIDTO(game.getAITypes());
     }
```

Comment:
Make an AIManager class that handles logic for AIs, then put `getAITypes()` in that class